.github/workflows/static-checks-mypy-docs.yml

```diff
@@ -96,6 +96,19 @@ on:  # yamllint disable-line rule:truthy
         description: "Whether to build docs (true/false)"
```

Comment:
Should i change this file name ? `ci-image-checks.yml` ?

Reply:
yep